### PR TITLE
[EXP-2008] Add decision brief for rollouts not affected by "studies" preference

### PR DESCRIPTION
**Pull request by travis79**

### CODE REVIEW

File: docs/adr/0006-treat-rollouts-differently.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Currently, rollouts are ignored if the users has opted out of studies because Nimbus treats rollouts the same as studies, obeying the user preference settings for experiments.
+
+## Decision Outcome
+
+Because of the differences between rollouts and experiments, the Nimbus team would like to treat them independently from each other.
+
+This would include:
+- Disconnecting rollouts from the "studies" application preference so that rollouts could still be delivered to users who have otherwise opted out of participating in experiments.
+- To preserve user-choice, we will require a new preference in applications that integrate Nimbus. This preference will enable the user to opt-out of rollouts, if desired. The status of this preference will be communicated to Nimbus at initialization and if it changes during runtime.
```
Comment (jhugman) [inline docs/adr/0006-treat-rollouts-differently.md:29]: Is this necessary? The user-choice at the moment is conflated with experiments. By adding an extra layer of user choice we are increasing our support costs.

File: docs/adr/0006-treat-rollouts-differently.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Configuring different settings for a feature for different audiences remotely
+- A "kill switch" if you want to launch a feature but then turn it off if something goes wrong.
+
+Currently, rollouts are ignored if the users has opted out of studies because Nimbus treats rollouts the same as studies, obeying the user preference settings for experiments.
+
+## Decision Outcome
+
+Because of the differences between rollouts and experiments, the Nimbus team would like to treat them independently from each other.
+
+This would include:
```
Comment (jhugman) [inline docs/adr/0006-treat-rollouts-differently.md:27]: Does this also include relaxing the constraint that we can only support one rollout out per feature at a time?

File: docs/adr/0006-treat-rollouts-differently.md (hunk 1 above)
Comment (travis79) [reply docs/adr/0006-treat-rollouts-differently.md:29]: I don't know if it is necessary or not. From a user's perspective I would say "yes, I would like to have the choice to determine when changes are made", but from an operational standpoint I can see where not having a preference would be simpler.

File: docs/adr/0006-treat-rollouts-differently.md (hunk 2 above)
Comment (travis79) [reply docs/adr/0006-treat-rollouts-differently.md:27]: I hadn't considered that for this particular brief, I was only focusing on the subject of whether we treat rollouts the same in regards to the opt-out preference for studies. In my opinion, this makes sense to treat rollouts differently and not block the feature with them

File: docs/adr/0006-treat-rollouts-differently.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## Context and Problem Statement
+
+Rollouts are a feature of Nimbus and are distinct from experiments in that they are not measurement instruments but instead represent off-train configuration updates.
+
+Rollouts can serve many purposes:
+- Launching a winning branch of an experiment faster than the trains.
+- Launching a configuration to non-experiment users during an experiment after a short period of verification
+- Configuring different settings for a feature for different audiences remotely
+- A "kill switch" if you want to launch a feature but then turn it off if something goes wrong.
+
```
Comment (travis79) [inline docs/adr/0006-treat-rollouts-differently.md:20]: Self-review: note to add information about the messaging requirement for being able to send messages to users who have opted out of experiments.

File: docs/adr/0006-treat-rollouts-differently.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Opting out of studies should not affect rollouts
+
+> Status: Open (Pending input from Trust & Legal regarding the addition of a new preference)
+
+> Author: Travis Long
+
+> Deciders: Nimbus team, Trust & Legal
```
Comment (jaredlockhart) [inline docs/adr/0006-treat-rollouts-differently.md:7]: Should this also include any of the following?
* Desktop Firefox product team members
* Mobile Firefox product team members
* Release management
* Data team members

**Comment by jaredlockhart** [comment]
Thank you @travis79 this is very concise 🙏

File: docs/adr/0006-treat-rollouts-differently.md (hunk 4 above)
Comment (travis79) [reply docs/adr/0006-treat-rollouts-differently.md:7]: Yes, those are all probably reasonable stakeholders affected by this. I'll add them to the decider list

**Comment by travis79** [comment]
Merging this with updated information after talking to Trust & Legal. If it requires updates or changes I'll be happy to open a new PR

**Pull request merged**